packages/cmrjs/tests/cmr-utils/test-reconcilecmrmetadata.js

```diff
@@ -106,10 +143,25 @@ test('reconcileCMRMetadata calls updateEcho10XMLMetadata but not publishECHO10XM
   const restorePublishECHO10XML2CMR = cmrUtils.__set__('publishECHO10XML2CMR', fakePublishECHO10XML2CMR);
 
   // act
-  await cmrUtils.reconcileCMRMetadata(granId, updatedFiles, distEndpoint, published);
+  await cmrUtils.reconcileCMRMetadata({
+    granuleId: granId,
+    updatedFiles,
+    backendUrl,
+    distEndpoint,
+    published
+  });
+
+  const paramsIntoUpdateEcho10XML = {
+    cmrFile: updatedFiles[1],
+    files: updatedFiles,
+    backendUrl,
+    distEndpoint,
+    buckets: new BucketsConfig(fakeBuckets)
+  };
 
   // assert
-  t.true(fakeUpdateCMRMetadata.calledOnceWith(updatedFiles[1], updatedFiles, distEndpoint));
+  t.deepEqual(paramsIntoUpdateEcho10XML, fakeUpdateCMRMetadata.firstCall.args[0]);
```

Comment:
I'm not happy about this solution. I was unable to get `calledOnceWith` to work with a parameter object as input. I think _may_ be because of the `BucketsConfig` object isn't the exact same object?